java/helidon-mp/src/main/java/web/helidon/IndexResource.java

```diff
@@ -0,0 +1,16 @@
+package web.helidon;
+
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+
+@Path("/")
+public class IndexResource {
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
```

Comment:
should be text plain 

If you sport any error if other implementations, do not hesitate to PR